cmd/crictl/container.go

```diff
@@ -326,6 +328,11 @@ var removeContainerCommand = &cli.Command{
 			Aliases: []string{"f"},
 			Usage:   "Force removal of the container, disregarding if running",
 		},
+		&cli.BoolFlag{
+			Name:    "keep-logs",
+			Aliases: []string{"kl"},
```

Comment:
```suggestion
			Aliases: []string{"k"},
```

I'm not a huge fan of 2 char aliases because it could also mean `-k -l`.